scripts/start_civ.sh

```diff
@@ -71,6 +71,19 @@ GUEST_STATIC_OPTION="\
 
 
 #------------------------------------------------------         Functions       ----------------------------------------------------------
+function check_non_graphical_target() {
+    local target=$(/bin/systemctl get-default)
+    if [[ $target  == *"graphical.target"* ]]; then
+        return -1
+    fi
+}
+
+function ask_set_non_graphical_target() {
+    echo "System is still in graphical.target. Execute below command to set as multi-user targeti, and reboot before launching GVT-d."
```

Comment:
why not print and exit in check_non_graphical_target()? no need a extra function.

Reply:
updated